Formula/mmctl.rb

```diff
@@ -0,0 +1,27 @@
+class Mmctl < Formula
+  desc "Remote CLI tool for Mattermost server"
+  homepage "https://github.com/mattermost/mmctl"
+  url "https://github.com/mattermost/mmctl/archive/0.2.1.tar.gz"
+  sha256 "a9922c893b479b4846ee6dc8bf6c98555562828bd0b2d935cbde21f9b6374103"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOBIN"] = buildpath/bin
+    ENV["ADVANCED_VET"] = "FALSE"
+    ENV["BUILD_HASH"] = "c330b36f679b69ac33fa1e561190ad2eb3466777"
```

Comment:
I'm afraid it's easy to forget to update `ENV["BUILD_HASH"]` in next releases. I would recommend either clone repo and get hash from git (see [`consul-template`](https://github.com/Homebrew/homebrew-core/blob/master/Formula/consul-template.rb)) or use fake one like "homebrew" (see [`scw`](https://github.com/Homebrew/homebrew-core/blob/master/Formula/scw.rb)). I'm in favour of using the first one.